app/models/order.rb

```diff
@@ -35,7 +35,12 @@ def ship_to_names
   end
 
   def to_json
-    { id: id, status: status, order_details: order_details.sort_by(&:id).map(&:to_json) }.to_json
+    {
+      id: id,
+      status: status,
+      order_details: order_details.sort_by(&:id).map(&:to_json),
+      in_requested_status: !!in_requested_status? # rubocop:disable Style/DoubleNegation
```

Comment:
why do we need to double bang if this method returns a boolean with .include?
```
Returns true if the given object is present in self (that is, if any element == object), otherwise returns false.
```